packages/@o3r/rules-engine/src/engine/operator/operators/date-based.operators.ts

```diff
@@ -16,6 +16,40 @@ export const inRangeDate: Operator<Date, [DateInput, DateInput], DateInput> = {
   validateRhs: isValidDateRange
 };
 
+/**
+ * Check if the value of the variable is in the next x minutes
+ *
+ * @title is in next minutes
+ */
+export const dateInNextMinutes: Operator<Date, [DateInput, number], DateInput> = {
+  name: 'dateInNextMinutes',
+  evaluator: (leftDateInput, [rightDate, minutes]) => {
+    const targetDate = new Date(rightDate);
+    const leftDate = new Date(leftDateInput);
+    targetDate.setMinutes(targetDate.getMinutes() + +minutes);
+    return leftDate <= targetDate;
+  },
+  validateLhs: isValidDateInput,
+  validateRhs: isValidDateWithMinutes
+};
+
+/**
+ * Check if the value of the variable is not in the next x minutes
+ *
+ * @title is not in next minutes
+ */
+export const dateNotInNextMinutes: Operator<Date, [DateInput, number], DateInput> = {
+  name: 'dateNotInNextMinutes',
+  evaluator: (leftDateInput, [rightDate, minutes]) => {
+    const targetDate = new Date(rightDate);
+    const leftDate = new Date(leftDateInput);
+    targetDate.setMinutes(targetDate.getMinutes() + +minutes);
```

Comment:
Not sure this is correct, you should add the minutes to the left date and ensure that the left date is still inferior to the right one.
(Maybe a UT testing the calcul would help)

Reply:
to clarify: 
`inNextMinutes` returns:
- `false` for date before `now` and for dates after `now` + `nextMinutes`
- `true` for dates between `now and `now` + `nextMinutes`

`notInNextMinutes returns:
- `false` for date before `now` and for dates between `now and `now` + `nextMinutes`
- `true` for dates after `now` + `nextMinutes`